CHANGELOG.md

```diff
@@ -1,6 +1,28 @@
 NativeScript CLI Changelog
 ================
 
+1.2.2 (2015, August 7)
+==
+
+### New
+* [Implemented #470](https://github.com/NativeScript/nativescript-cli/issues/470): Add timeout option to debug command.
+* [Implemented #682](https://github.com/NativeScript/nativescript-cli/issues/682): Set `--debug-brk` as default option of `tns debug` command.
```

Comment:
the default option of `tns debug`

And drop command 

Btw is this for tns debug or for tns debug android